akka-stream/src/main/scala-jdk9-only/akka/stream/scaladsl/JavaFlowSupport.scala

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com/>
+ */
+
+package akka.stream.scaladsl
```

Comment:
notice the directory, it's `akka-stream/src/main/scala-jdk9-only/akka/stream/scaladsl/JavaFlowSupport.scala`
which is different than the `scala-jdk8` which style I had reserved for the multi-release jars... But feel free to suggest better names